src/variables/hint.js

```diff
@@ -42,13 +42,25 @@ import './mode';
 CodeMirror.registerHelper('hint', 'graphql-variables', (editor, options) => {
   const cur = editor.getCursor();
   const token = editor.getTokenAt(cur);
+
+  const results = getVariablesHint(cur, token, options);
+  if (results.list && results.list.length > 0) {
+    results.from = CodeMirror.Pos(results.from.line, results.from.column);
+    results.to = CodeMirror.Pos(results.to.line, results.to.column);
+  }
+  CodeMirror.signal(editor, 'hasCompletion', editor, results, token);
```

Comment:
I'm curious if we need to call this if `list.length === 0`. Could it be included into the condition above without breaking anything?